test/modellib/ModelLib.cpp

```diff
@@ -134,7 +134,11 @@ bool ModelLibBuilder::areCloseFloat(
     const OMTensor *res, const OMTensor *ref) const {
   if (!res || !ref)
     return false;
-  float rtol = getenv("TEST_RTOL") ? atof(getenv("TEST_RTOL")) : 1e-5;
+  // Because of floating point impact of using FMA instead of multiply followed
+  // by an add,the default RTOL is increased from 1e-5 to 5e-5. This bound has
```

Comment:
[nit] missing space  "add,the" -> "add, the"

Reply:
tx